README.md

```diff
@@ -96,7 +97,7 @@ No hybrid signature algorithms are currently implemented. If those are needed fo
 
 ## Quickstart
 
-We regularly test the fork on Ubuntu 18.04 and above. Support for other platforms and operating systems, as well as for old versions of GCC (< 8) and Clang (< 8), is not guaranteed.
```

Comment:
Why not keep the caveat for other platforms and OSs?

Reply:
It's mentioned in Quickstart :) https://github.com/pi-314159/boringssl/blob/20240302-X25519/README.md#quickstart

GCC/Clang/etc: Versions that are shipped in the release should work

Reply:
I don’t see the necessity to state this explicitly. But I can add it back if it’s needed.

Reply:
It would set proper user expectations (fewer questions/issues raised in this regard). Your call really.